course/frontend/arrays/scripts/main.js

```diff
@@ -213,3 +213,76 @@ function flat(arr, newArray) {
 
 flat([1, 2, 3, [4, 5], 6]);
 flat([1, 2, [3, 4, [1, 2, [1, 2, 3]]], 5, 6]);
+
+
+// Exercises 28, 29
+
+/*
+### unionArray
+
+Write a JavaScript program to compute the union of two arrays.
+Sample Data :
+
+    console.log(unionArray([1, 2, 3], [100, 2, 1, 10]));
+    [1, 2, 3, 10, 100]
+
+### removeFalsy
+
+Write a JavaScript function to remove. 'null', '0', '""', 'false', 'undefined' and 'NaN' values from an array.
+
+    Sample array : [NaN, 0, 15, false, -22, '',undefined, 47, null]
+    Expected result : [15, -22, 47]
+
+*/
+
+/** 
+ * Function unionArray 
+ * Returns one numeric merged array without duplicated values.
+ * @param {array} first argument
+ * @param {array} second argument
+ * @returns {array} numeric values.
+ */
+ 
+ /**
+ * Function removeFalsy
+ * Returns one numeric array without empty,
+ * @param {array} first argument 
+ * @returns {array}  
+ */
+
+var unionArray, removeFalsy;
+
+(function(){
+'use strict'
```

Comment:
ADD ";" at the end of this statement (we are in ES5 ,)

Reply:
yes, you are right. I did not notice about this. A